superset-frontend/src/dashboard/components/OverwriteConfirm/OverwriteConfirmModal.tsx

```diff
@@ -0,0 +1,212 @@
+import Button from 'src/components/Button';
+import { DashboardState } from 'src/dashboard/types';
+import {
+  saveDashboardRequest,
+  setOverrideConfirm,
+} from 'src/dashboard/actions/dashboardState';
+import { t, styled } from '@superset-ui/core';
+import { SAVE_TYPE_OVERWRITE_CONFIRMED } from 'src/dashboard/util/constants';
+
+const STICKY_HEADER_TOP = 16;
+const STICKY_HEADER_HEIGHT = 32;
+
+const StyledTitle = styled.h2`
+  ${({ theme }) => `
+     color:  ${theme.colors.grayscale.dark1}
+   `}
+`;
+
+const StyledEditor = styled.div`
+  ${({ theme }) => `
+     table {
+       border: 1px ${theme.colors.grayscale.light2} solid;
+     }
+     pre {
+       font-size: 11px;
+       padding: 0px;
+       background-color: transparent;
+       border: 0px;
+       line-height: 110%;
+     }
+   `}
+`;
+
+const StackableHeader = styled(Button)<{ top: number }>`
+  ${({ theme, top }) => `
+     position: sticky;
+     top: ${top}px;
+     background-color: ${theme.colors.grayscale.light5};
+     margin: 0px;
+     padding: 8px 4px;
+     z-index: 1;
+     border: 0px;
+     border-radius: 0px;
+     width: 100%;
+     justify-content: flex-start;
+     border-bottom: 1px ${theme.colors.grayscale.light1} solid;
+     &::before {
+       display: inline-block;
+       position: relative;
+       opacity: 1;
+       content: "\\00BB";
+     }
+   `}
+`;
+
+const StyledBottom = styled.div<{ inView: boolean }>`
+  ${({ inView }) => `
+     margin: 8px auto;
+     text-align: center;
+     opacity: ${inView ? 0 : 1};
+  `}
+`;
+
+type Props = {
+  overwriteConfirmMetadata: DashboardState['overwriteConfirmMetadata'];
+};
+
+const OverrideConfirmModal = ({ overwriteConfirmMetadata }: Props) => {
+  const [bottomRef, hasReviewed] = useInView({ triggerOnce: true });
+  const dispatch = useDispatch();
+  const onHide = useCallback(
+    () => dispatch(setOverrideConfirm(undefined)),
+    [dispatch],
+  );
+  const anchors = useMemo<RefObject<HTMLDivElement>[]>(
+    () =>
+      overwriteConfirmMetadata
+        ? overwriteConfirmMetadata.overwriteConfirmItems.map(() =>
+            createRef<HTMLDivElement>(),
+          )
+        : [],
+    [overwriteConfirmMetadata],
+  );
+  const onAnchorClicked = useCallback(
+    (index: number) => {
+      anchors[index]?.current?.scrollIntoView({ behavior: 'smooth' });
+    },
+    [anchors],
+  );
+  const onConfirmOverwrite = useCallback(() => {
+    if (overwriteConfirmMetadata) {
+      dispatch(
+        saveDashboardRequest(
+          overwriteConfirmMetadata.data,
+          overwriteConfirmMetadata.dashboardId,
+          SAVE_TYPE_OVERWRITE_CONFIRMED,
+        ),
+      );
+    }
+  }, [dispatch, overwriteConfirmMetadata]);
+
+  return (
+    <Modal
+      width="auto"
+      height="50vh"
+      show={Boolean(overwriteConfirmMetadata)}
+      title={t('Confirm overwrite')}
+      footer={
+        <>
+          {t('* Scroll down to the bottom to complete the review. ')}
```

Comment:
```suggestion
          {t('Scroll down to the bottom to complete the review. ')}
```

Not sure if `*` is needed.